lib/matplotlib/tests/test_axes.py

```diff
@@ -8195,3 +8195,13 @@ def test_bar_leading_nan():
         for b in rest:
             assert np.isfinite(b.xy).all()
             assert np.isfinite(b.get_width())
+
+
+@image_comparison(["extent_units.pdf"])
+def test_extent_units():
+    mpl.style.use("mpl20")
+    fig, ax = plt.subplots()
+    dates = np.arange("2020-01-01", "2020-01-13", dtype='datetime64')
```

Comment:
Again, why the `arange` if you are only using two values?

Reply:
fixed it